README.md

```diff
@@ -124,8 +124,13 @@ GRANT ALL PRIVILEGES ON otptest.* TO otptest@localhost;
 CREATE USER otptest2@localhost IDENTIFIED BY 'otptest2';
 GRANT ALL PRIVILEGES ON otptest.* TO otptest2@localhost;
 
+# in MySQL < 5.7, REQUIRE SSL must be given in GRANT
```

Comment:
Good comments.

Is this MySQL-specific comment syntax? In standard SQL, the comments start with `--`. I've only used the standard one; never bothered to learn the non-standard one(s).

Reply:
Uh, looks like it is: https://dev.mysql.com/doc/refman/8.0/en/comments.html

I for my part wasn't aware of `#` being MySQL-specific, so I used it without thinking.

Reply:
Opened another PR #159 to use the Standard SQL style.

Reply:
Then you've learned something. :+1:

Reply:
True :)